projects/storefrontlib/cms-components/myaccount/my-interests/my-interests.component.ts

```diff
@@ -94,10 +94,7 @@ export class MyInterestsComponent implements OnInit, OnDestroy {
     this.sortLabels = this.getSortLabels();
   }
 
-  private getSortLabels(): Observable<{
-    byNameAsc: string;
-    byNameDesc: string;
-  }> {
+  private getSortLabels(): Observable<{ [key: string]: string }> {
```

Comment:
What's the purpose of this change?  (the method is prvate, but it has impact on a public property)